samples/app-anonymous-users/nodejs/README.md

```diff
@@ -0,0 +1,295 @@
+  4. Under **Manage**, select **Expose an API**. 
+  5. Select the **Set** link to generate the Application ID URI in the form of `api://{base-url}/botid-{AppID}`. Insert your fully qualified domain name (with a forward slash "/" appended to the end) between the double forward slashes and the GUID. The entire ID should have the form of: `api://fully-qualified-domain-name/botid-{AppID}`
+      * ex: `api://%ngrokDomain%.ngrok.io/botid-00000000-0000-0000-0000-000000000000`.
+  6. Select the **Add a scope** button. In the panel that opens, enter `access_as_user` as the **Scope name**.
+  7. Set **Who can consent?** to `Admins and users`
+  8. Fill in the fields for configuring the admin and user consent prompts with values that are appropriate for the `access_as_user` scope:
+      * **Admin consent title:** Teams can access the user’s profile.
+      * **Admin consent description**: Allows Teams to call the app’s web APIs as the current user.
+      * **User consent title**: Teams can access the user profile and make requests on the user's behalf.
+      * **User consent description:** Enable Teams to call this app’s APIs with the same rights as the user.
+  9. Ensure that **State** is set to **Enabled**
+  10. Select **Add scope**
+      * The domain part of the **Scope name** displayed just below the text field should automatically match the **Application ID** URI set in the previous step, with `/access_as_user` appended to the end:
+          * `api://[ngrokDomain].ngrok.io/botid-00000000-0000-0000-0000-000000000000/access_as_user.
+  11. In the **Authorized client applications** section, identify the applications that you want to authorize for your app’s web application. Each of the following IDs needs to be entered:
+      * `1fec8e78-bce4-4aaf-ab1b-5451cc387264` (Teams mobile/desktop application)
+      * `5e3ce6c0-2b1f-4285-8d4b-75ee78787346` (Teams web application)
+  12. Navigate to **API Permissions**, and make sure to add the follow permissions:
+  -   Select Add a permission
+  -   Select Microsoft Graph -\> Delegated permissions.
+      - `User.Read` (enabled by default)
+  -   Click on Add permissions. Please make sure to grant the admin consent for the required permissions.
+  13. Navigate to **Authentication**
+      If an app hasn't been granted IT admin consent, users will have to provide consent the first time they use an app.
+  - Set a redirect URI:
+      * Select **Add a platform**.
+      * Select **Single-page application**.
+      * Enter the **redirect URI** for the app in the following format: `https://{Base_Url}/auth-end` and `https://{Base_Url}/auth-start`.
+  14.  Navigate to the **Certificates & secrets**. In the Client secrets section, click on "+ New client secret". Add a description(Name of the secret) for the secret and select “Never” for Expires. Click "Add". Once the client secret is created, copy its value, it need to be placed in the appsettings.json.
```

Comment:
I think you meant the .env file (not appsettings.json).

Reply:
Fixed